tests/unit/lib/hotkeyManager.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { it } from "vitest";
+import { HotkeyManager, HotkeysJs } from "@/plugins/hotkeyPlugin";
+
+type DummyHotkeysJs = HotkeysJs & {
+  registeredHotkeys: {
+    key: string;
+    scope: string;
+    // callbackを持たせると比較が面倒になるので持たせない
+    // callback: (e: KeyboardEvent) => void;
+  }[];
+  currentScope: string;
+};
+const createHotkeyManager = (): {
+  hotkeyManager: HotkeyManager;
+  dummyHotkeysJs: DummyHotkeysJs;
+} => {
+  const registeredHotkeys: DummyHotkeysJs["registeredHotkeys"] = [];
+  const dummyHotkeysJs = ((
+    key: string,
+    { scope }: { scope: string },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    callback: (e: KeyboardEvent) => void
+  ) => {
+    registeredHotkeys.push({ key, scope });
+  }) as DummyHotkeysJs;
+  dummyHotkeysJs.unbind = (key: string) => {
+    const index = dummyHotkeysJs.registeredHotkeys.findIndex(
+      (h) => h.key === key
+    );
+    if (index !== -1) {
+      registeredHotkeys.splice(index, 1);
+    }
+  };
+  dummyHotkeysJs.setScope = (scope: string) => {
+    dummyHotkeysJs.currentScope = scope;
+  };
+  dummyHotkeysJs.registeredHotkeys = registeredHotkeys;
+  dummyHotkeysJs.currentScope = "talk";
+  return { hotkeyManager: new HotkeyManager(dummyHotkeysJs), dummyHotkeysJs };
+};
+
+// @ts-expect-error これがないとログを吐けずにエラーを吐く
+window.electron = {
+  logInfo: console.log,
+};
+
+it("registerできる", () => {
+  const { hotkeyManager } = createHotkeyManager();
+  hotkeyManager.register({
+    editor: "talk",
+    name: "音声書き出し",
+    callback: () => {
+      /* noop */
+    },
+  });
+});
+
```

Comment:
↑２つのエラーを足した状態で↓のテスト実行すると落ちました。
実際にショートカット登録つけたり消したりしても再現する･･･はず？
```suggestion

it("registerしたあとloadしてreplaceできる", () => {
  const { hotkeyManager } = createHotkeyManager();
  const name = "音声書き出し";
  hotkeyManager.register({
    editor: "talk",
    name: name,
    callback: () => {
      /* noop */
    },
  });
  hotkeyManager.load([{ action: name, combination: "1" }]);
  hotkeyManager.replace({ action: name, combination: "2" });
  hotkeyManager.replace({ action: name, combination: "" });
  hotkeyManager.replace({ action: name, combination: "3" });
});
```